pkg/commands/git_commands/commit.go

```diff
@@ -45,7 +45,7 @@ func (self *CommitCommands) AddCoAuthor(sha string, value string) error {
 		return err
 	}
 
-	message = message + fmt.Sprintf("\nCo-authored-by: %s", value)
+	message = message + fmt.Sprintf("\n\nCo-authored-by: %s", value)
```

Comment:
I think we could be smarter here, and add only one `\n` instead of two if the previous line already is `Co-authored-by:`, for the case that you were pairing on the commit with multiple people.

Reply:
I did some tests with multiple authors, the current behaviour is working as expected

Reply:
What _is_ the expected behaviour, though? `A` or `B`?

A)
```
This commit improves the performance by a whopping 5000%.

Co-authored-by: Programmer 1 <mail@mail.com>

Co-authored-by: Programmer 2 <mail@mail.com>

Co-authored-by: Programmer 3 <mail@mail.com>
```

B)
```
This commit improves the performance by a whopping 5000%.

Co-authored-by: Programmer 1 <mail@mail.com>
Co-authored-by: Programmer 2 <mail@mail.com>
Co-authored-by: Programmer 3 <mail@mail.com>
```

I tested it now and currently `A` is implemented.

Reply:
Yes, it's clearly B for me, as I said above. I'm not sure it's worth the effort to change this, since it's probably not a very common situation to have multiple co-authors, but it's not super hard to do either, so why not.

I started working on changing this quite a while ago, along with a few other small tweaks to the branch, but I got side-tracked and it fell through the tracks. I'm hoping to make more progress on this on the weekend.